Bottom: c58710aa09f69bf22e6c628588c00895879b9ba2
Top:    fb5a63589982eb311e6938f508182956d616fe8f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 15:03:27 +0100

doc: Remove execute permissions on Makefile.am and gs/Makefile.am


---

diff --git a/doc/Makefile.am b/doc/Makefile.am
old mode 100755
new mode 100644
diff --git a/doc/gs/Makefile.am b/doc/gs/Makefile.am
old mode 100755
new mode 100644
